foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -162,7 +162,7 @@ Writing an HTML comment is simple: We just enclose the comment with `<!--` and `
 
  **VSCode keyboard shortcut** 
 
-The following shortcut will create a new comment, convert any line to a comment, or uncomment any line:
+If you find tying out the Comments syntax tiring, the following shortcut will help you quickly create a new comment, convert any line to a comment, or uncomment any line:
```

Comment:
Thaks for the changes. It does seem to have a typo:

"tying" should be "typing" I think

Reply:
Done